tests/integration/targets/vm/tasks/02_shutdown.yml

```diff
@@ -1,44 +1,43 @@
 ---
 # This is a part of the vm module; testing shutdown logic.
-# The main vm test was devided into smaller tests for sake of synchronous testing.
+# The main vm test was divided into smaller tests for sake of synchronous testing.
+# Question: do we want to test nice ACPI shutdown, or force shutdown, or both?
+# We should test correct force shutdown too.
```

Comment:
Maybe we can test both shutdowns here in file 02.yml
After that we can divide VM test into two separate folders "shutdown" and "main-operations" something like that.
We can divide even further if necessary. But this should already create two ~10 min tests instead of one that takes ~20 min.